### ISSUE TITLE
representativeness assumption

### ISSUE THREAD

**Issue by davidwhogg** [AnnieJumpCannon/RAVE#19]
We assume that the training data are spectrally similar to the test data where they overlap in label space, and we assume that the training data spans enough of the label space to capture the variation in the test-set spectra.

Or something like that? It isn't that the training set is "representative", because that might imply a similar distribution in logg, Teff.
